src/utils/partition_stl_containers.hpp

```diff
@@ -0,0 +1,83 @@
+//========================================================================================
+// (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+//
+// This program was produced under U.S. Government contract 89233218CNA000001 for Los
+// Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+// for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+// in the program are reserved by Triad National Security, LLC, and the U.S. Department
+// of Energy/National Nuclear Security Administration. The Government is granted for
+// itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works, distribute copies to
+// the public, perform publicly and display publicly, and to permit others to do so.
+//=======================================================================================
+#ifndef UTILS_PARTITION_STL_CONTAINERS_HPP_
+#define UTILS_PARTITION_STL_CONTAINERS_HPP_
+
+#include <string>
+#include <vector>
+
+#include "error_checking.hpp"
+
+namespace parthenon {
+namespace Partition {
```

Comment:
lower-case to match root namespace
```suggestion
namespace partition {
```

Reply:
So I was actually unsure about this one. I know we've been using lower-case namespaces... but the reason I created a namespace at all is because it felt like a nice shorthand `Partition::RestOfTheSentence`. And that felt like it should be capitalized because that's how we've been doing it for functions.

That said, I can see how it's a style violation, so I'll change it.